MethodsHomework/MethodsHomework.cs

```diff
@@ -166,8 +169,136 @@ public class YourAnswers
     {
         //READ ME
         // You write all your methods here.
+        public static int DoNothing()
+        {
+            return 0 ;
+        }
+
+        public static int ReturnIntWithValue25()
+        {
+            return 25;
+        }
+
+        public static int AddTwoNumbers(int x, int y)
+        {
+            int addIt = x + y;
+            return addIt;
+        }
+
+        public static int IncrementMe(int x)
+        {
+            int y = x++;
```

Comment:
We haven't gone into detail on the pre-increment (++x) and post-increment (x++) operators, other than to note that they modify the value of x. Using the increment operator and assigning the result into a second variable can be confusing.

If you change to pre-increment (`int y = ++x`), that would work. But if you're using either x++ or ++x, I'd recommend just putting that on its own line, with no assignment/no other variable.